go.mod

```diff
@@ -14,30 +15,9 @@ require (
 	volcano.sh/apis v1.2.0-k8s1.19.6
 )
 
-
```

Comment:
Where did this change come from?

Reply:
Since we move to volcano.sh/api and these statements are no longer needed. Should I  move these changes to another change set?

Reply:
Yes separate PRs would be better.

Reply:
OK. I've amended the commit.